private-templates-service/client/src/components/Designer/Designer.tsx

```diff
@@ -26,8 +26,8 @@ const mapStateToProps = (state: RootState) => {
 
 const mapDispatchToProps = (dispatch: any) => {
   return {
-    updateTemplate: (templateID: string, currentVersion: string, templateJSON: string, templateName: string, sampleDataJSON: string) => {
-      dispatch(updateTemplate(templateID, undefined, templateJSON, templateName, sampleDataJSON));
+    updateTemplate: (templateID: string, currentVersion: string, templateJSON: object, templateName: string, sampleDataJSON: object) => {
+      dispatch(updateTemplate(templateID, undefined, templateJSON, sampleDataJSON, templateName));
```

Comment:
order of variables changed? is this intended?

Reply:
Oops, that's an accident. Done.